apps/api/lib/api/job/update_game.ex

```diff
@@ -9,16 +9,22 @@ defmodule API.Job.UpdateGame do
     job = %Verk.Job{
       queue: :default,
       class: __MODULE__,
-      args: [game.id, %{round: game.round + 1}]
+      args: [game.id, game.round]
     }
 
     Verk.schedule(job, perform_at(game))
   end
 
-  def perform(id, attrs) do
-    id
-    |> DB.Game.get
-    |> DB.Game.update(attrs)
+  def perform(id, round) do
+    game = DB.Game.get(id)
+
+    if game.round == round do
+      selected_player = API.Lottery.select(id)
```

Comment:
This (or in `Lottery` itself) might be a good place to introduce some logging

Reply:
👍  yea we should do a logging and monitoring audit

Reply:
https://trello.com/c/EkrptCRy/53-add-logging-and-monitoring-tools